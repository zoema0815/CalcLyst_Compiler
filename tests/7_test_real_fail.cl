int main () {
    Real c;
    Imaginary d;
    Complex z;
    Real test_real;
    c = 3.7r;
    d = 3.2i;
    z = d~c;
    test_real = z; /* Invalid to assign a real as a complex number */
    return 0;
}