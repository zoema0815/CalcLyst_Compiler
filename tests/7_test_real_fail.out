Fatal error: exception Failure("illegal assignment Real = Complex in test_real = z")
